OpenRA.Game/Settings.cs

```diff
@@ -408,7 +408,7 @@ public static string SanitizedServerName(string dirty)
 		{
 			var clean = SanitizedName(dirty);
 			if (string.IsNullOrWhiteSpace(clean))
-				return new ServerSettings().Name;
+				return $"{Environment.UserName}'s Game";
```

Comment:
Announcing a valid user account name (which has a good chance of being accessible via `ssh` for dedicated servers) to the public internet seems particularly unwise.

Reply:
I have only been using Amazon Web Services lately, where you disallow SSH for everything but your home address to avoid getting DDOSed by people trying out common usernames. My preferred solution would be `UserPrincipal.GivenName`, but that will only work on Windows. I set it to the computer's name now.

Reply:
This now isn't *quite* as bad, but it is unneccessary and a bad idea to be leaking any state from the current user environment.

Reply:
What do you suggest instead?

Reply:
“OpenRA Server”, or, if you really do insist on hiding remaining references to OpenRA from your UI then “$modname Server”. I thought that this would have been the obvious choice.

Reply:
I was spammed a bit with issues, so I didn't read that one properly. Sorry. I thought @0ad would also query the OS username. The player name configured in OpenRA is a great idea, because on first game start you get this nice dashboard you created, so chances are high that it will get changed and we don't end up with `Commander's Game` stepping up the ranks instead.